honeybee_energy/programtype.py

```diff
@@ -34,35 +57,11 @@ class ProgramType(object):
     """
     __slots__ = ('_name', '_people', '_lighting', '_electric_equipment',
                  '_gas_equipment', '_infiltration', '_ventilation',
-                 '_setpoint', '_locked')
+                 '_setpoint', '_locked)
```

Comment:
The reason why you get an error is because you accidentally deleted a `'` here.

Reply:
Thanks @chriswmackey for that, I guess I catched it once I realized that the mistake could be mine and not a dependency issue,,,sorry the extra work! 
Ironing out other bugs in the docs now...